cmd/analyzer/analyzer.go

```diff
@@ -84,11 +107,32 @@ func parseFlags() cliFlags {
 	if cli.shardPrefixesFlag != "" {
 		cli.shardPrefixes = strings.Split(cli.shardPrefixesFlag, ",")
 	}
+
+	if cli.json {
+		data := &MistConfig{
+			Name:         "LivepeerAnalyzer",
+			Version:      version,
+			Description:  "",
+			FriendlyName: "Livepeer Analyzer",
+			Optional:     []MistOptional{},
+		}
+		fs.VisitAll(func(f *flag.Flag) {
+			data.Optional = append(data.Optional, MistOptional{
+				Name:    f.Name,
+				Help:    f.Usage,
+				Default: f.DefValue,
+			})
+		})
+		b, _ := json.Marshal(data)
+		os.Stdout.Write(b)
+		os.Exit(255)
```

Comment:
Does mist require it to be an error exit code? Feels like a successful run if we print the json help and exit explicily. Could this be `0` instead?

Reply:
The existing mist binaries resolve the flag with a 255 exit code. I don't think that the exit code has anything to do with how it is being handled anyway, so it could be anything. I just kept 255 to follow existing behaviour.